activerecord/lib/active_record/connection_adapters/abstract_mysql_adapter.rb

```diff
@@ -141,6 +141,10 @@ def supports_datetime_with_precision?
         end
       end
 
+      def supports_virtual_columns?
+        !mariadb? && version >= "5.7.5"
```

Comment:
How about adding the comment (partially support for MariaDB) like `supports_index_sort_order?`?
https://github.com/rails/rails/blob/v5.0.1/activerecord/lib/active_record/connection_adapters/abstract_mysql_adapter.rb#L110-L114

I'm not sure there are similar DBs other than MariaDB.